packages/shared/src/modules/users/libs/validation-schemas/user-sign-in.validation-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from "zod";
+
+import { UserValidationMessage, UserValidationRule } from "../enums/enums.js";
+
+type UserSignInRequestValidationDto = {
+	email: z.ZodString;
+	password: z.ZodString;
+};
+
+const userSignIn = z
+	.object<UserSignInRequestValidationDto>({
+		email: z
+			.string()
+			.trim()
+			.min(UserValidationRule.EMAIL_MINIMUM_LENGTH, {
+				message: UserValidationMessage.EMAIL_REQUIRE,
```

Comment:
Shouldn't this be something like EMAIL_MIN_LENGTH instead of EMAIL_REQUIRE?

Reply:
Thanks, I missed that.